www/index.html

```diff
@@ -1614,6 +1632,28 @@
         </g>
         <path class="st20" d="M1105.5,904.1v40.5l58.2-33.3v-40.5L1105.5,904.1z M1162.7,908.8l-56.1,32.4v-36.6l56.1-32.4V908.8z"/>
     </g>
+    <g id="c33">
+        <g>
+            <g>
+                <polygon class="st52" points="1360.6,786.6 1351.8,791.7 1351.8,793 1360.6,787.9 					"/>
+                <polygon class="desk-left" points="1346,788.3 1351.8,791.7 1351.8,793 1346,789.6 					"/>
+                <polygon class="st53" points="1346,788.3 1351.8,791.7 1360.6,786.6 1354.8,783.2 					"/>
+            </g>
+        </g>
+        <g>
+            <g>
+                <polygon class="st52" points="1356.8,754.7 1350.8,758.2 1350.8,789.5 1356.8,786 					"/>
+                <polygon class="desk-left" points="1349.4,757.3 1350.8,758.2 1350.8,789.5 1349.4,788.7 					"/>
+                <polygon class="st19" points="1349.4,757.3 1350.8,758.2 1356.8,754.7 1355.4,753.9 					"/>
+            </g>
+        </g>
+        <g>
+            <polygon class="comp c28 available" points="1380.8,723.2 1322.6,756.5 1322.6,797.1 1380.8,763.8 				"/>
```

Comment:
```suggestion
            <polygon class="comp c33 available" points="1380.8,723.2 1322.6,756.5 1322.6,797.1 1380.8,763.8 				"/>
```
?

Reply:
Oh that should be `c33`. You're right. I'll open another PR.